docs/developer/getting-started.rst

```diff
@@ -29,7 +29,12 @@ To build and install the library:
 
   conda create -n scipp python=3.7
   conda env activate scipp
-  conda install # populate list here
+  conda install -c conda-forge appdirs numpy # ..etc. populate from meta.yml
+
+
+To build a debug version of the library:
```

Comment:
You don't specify the cmake_build_type in the following, won't this default to Release? https://github.com/scipp/scipp/blob/master/CMakeLists.txt#L39

Reply:
Good point. I assumed (without checking) that because we turned IPO and static linking off as part of getting a debug build. Updated that now.